honeycomb-guide/src/index.md

```diff
@@ -13,13 +13,38 @@ issues, thanks to the language's guarantees.
 
 ### Requirements
 
-- **Rust 1.75** - *The code may compile and work for earlier versions, but we do not test for those*
+- **Rust 1.76** - *The code may compile and work for earlier versions, but we do not test for those*
+
+### Quickstart
+
+#### Rust
+
+...
+
+#### Documentation
+
+You can generate this documentation locally using **mdbook** and **cargo doc**:
+
+```shell
+# Serve the doc on a local server
+mdbook serve --open -d ../target/doc/ honeycomb-guide/ &
+cargo doc --all --no-deps
+
+# Kill the local server
+kill $(pidof mdbook) 
+
+# Without pidof
+kill $(ps -e | awk '/mdbook/ {print $1}')
```

Comment:
```suggestion
```

Like in the readme. Can you have a way to not duplicate the readme and this file?